vagrant/config/versions.yaml

```diff
@@ -74,6 +74,16 @@ installers:
       - 'debian11'
       - 'ubuntu2004'
 
+  - foreman: '3.4'
+    katello: 'nightly'
```

Comment:
Please set this correctly

Reply:
Done.  I wasn't sure if I should use nightly or 4.6, due to the order of operations (i.e. use nightly first, then update when 4.6 is branched, or just use 4.6 from the start).